Earthfile

```diff
@@ -573,36 +595,44 @@ iso-remote:
     SAVE ARTIFACT /build/$ISO_NAME.iso.sha256 kairos.iso.sha256 AS LOCAL build/$ISO_NAME.iso.sha256
 
 netboot:
-   ARG OSBUILDER_IMAGE
-   FROM $OSBUILDER_IMAGE
-   COPY +version/VERSION ./
-   ARG VERSION=$(cat VERSION)
-   RUN echo "version ${VERSION}"
-   ARG ISO_NAME=${OS_ID}
-   ARG FROM_ARTIFACT
-   WORKDIR /build
-   ARG RELEASE_URL
-
-   COPY . .
-   IF [ "$FROM_ARTIFACT" = "" ]
-        COPY +iso/kairos.iso kairos.iso
-        RUN /build/scripts/netboot.sh kairos.iso $ISO_NAME $VERSION
-   ELSE
-        RUN /build/scripts/netboot.sh $FROM_ARTIFACT $ISO_NAME $VERSION
-   END
+    COPY +version/VERSION ./
+    RUN echo "version ${VERSION}"
+    ARG VERSION=$(cat VERSION)
+    ARG ISO_NAME=${OS_ID}-${VARIANT}-${FLAVOR}-${ARCH}-${MODEL}-${VERSION}
+    ARG OSBUILDER_IMAGE
+    FROM $OSBUILDER_IMAGE
+    ARG FROM_ARTIFACT
+    WORKDIR /build
+    ARG RELEASE_URL
+
+    COPY . .
+    IF [ "$FROM_ARTIFACT" = "" ]
+         COPY +iso/kairos.iso kairos.iso
+         RUN /build/scripts/netboot.sh kairos.iso $ISO_NAME $VERSION
+    ELSE
+         RUN /build/scripts/netboot.sh $FROM_ARTIFACT $ISO_NAME $VERSION
+    END
 
-   SAVE ARTIFACT /build/$ISO_NAME.squashfs squashfs AS LOCAL build/$ISO_NAME.squashfs
-   SAVE ARTIFACT /build/$ISO_NAME-kernel kernel AS LOCAL build/$ISO_NAME-kernel
-   SAVE ARTIFACT /build/$ISO_NAME-initrd initrd AS LOCAL build/$ISO_NAME-initrd
-   SAVE ARTIFACT /build/$ISO_NAME.ipxe ipxe AS LOCAL build/$ISO_NAME.ipxe
+    SAVE ARTIFACT /build/$ISO_NAME.squashfs squashfs AS LOCAL build/$ISO_NAME.squashfs
+    # TODO: why do the kernel and initrd files don't have an extension?
+    SAVE ARTIFACT /build/$ISO_NAME-kernel kernel AS LOCAL build/$ISO_NAME-kernel
+    SAVE ARTIFACT /build/$ISO_NAME-initrd initrd AS LOCAL build/$ISO_NAME-initrd
```

Comment:
why do the kernel and initrd files don't have an extension?

Reply:
what extension could they have? sometimes they migth have img or gz or img.gz so I guess its difficult to infer wha thte correct one should be.

also does it matter?

Reply:
No it doesn't I will update the docs then